release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -81,7 +81,7 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
     1. Compare the generated output to the new commits for the release to check if
        any notable change missed a release note.
     1. Reword release notes as needed, ideally in the original PRs so that the
-       release notes can be regnerated. Make sure to check notes for breaking
+       release notes can be regenerated. Make sure to check notes for breaking
```

Comment:
We should pull this from release-tools. See example here: https://github.com/kubernetes-csi/external-provisioner/pull/1183

Reply:
Done. Thanks for the lead!